### ISSUE TITLE
Ansible Playbook Error due to updated Pandevice Packages for vsys: Could not find schema node for xpath /config/devices/entry[@name='localhost.localdomain']/rulebase/security/rules

### ISSUE THREAD

**Issue by tush0618** [PaloAltoNetworks/pandevice#93]
Hi All,

So with the previous package of **Pandevice** _(available approx 2 months back)_, we are able to "add security rules" successfully to a PaloAlto Firewall via running **Ansible Playbook.** 
Because it was able to find/create below path using the pandevice packages: 
**/config/devices/entry[@name='localhost.localdomain']/vsys/entry[@name='vsys1']/rulebase/security/rules**

But after the new commits with respect to the vsys changes, its not able to create the above path. Its searching/creating the below path instead: 
**/config/devices/entry[@name='localhost.localdomain']/rulebase/security/rules**
  
and throwing error as :- 
`"msg": "Could not find schema node for xpath /config/devices/entry[@name='localhost.localdomain']/rulebase/security/rules`

Therefore, _vsys/entry[@name='vsys1']_ this part is missing.

Below is the sample playbook for adding security rule which is working for the older packages and not working for the newer package.

```
- hosts: paloalto
  connection: local

  vars:
    mgmt_ip: "10.***.**.**"
    admin_password: "admin"

  tasks:       
    # permit ssh to 1.1.1.1
    - name: permit ssh to 1.1.1.1
      panos_security_rule:
        ip_address: "{{ mgmt_ip }}"
        password: "{{admin_password}}"
        rule_name: 'SSH permit'
        description: 'SSH rule test'
        source_zone: ['UnTrustL-3']
        destination_zone: ['TrustL-3']
        source_ip: ['any']
        source_user: ['any']
        destination_ip: ['1.1.1.1']
        category: ['any']
        application: ['ssh']
        service: ['application-default']
        hip_profiles: ['any']
        action: 'allow'
        operation: 'add'
        commit: false

  roles:
    - role: PaloAltoNetworks.paloaltonetworks
```

Kindly look into the issue.

Regards,
Tushar

**Comment by btorresgil** [comment]
Thanks for opening this.  It's likely due to a change in how vsys works that was made in pandevice 0.5.0.  Any version prior to that would work as a workaround.  We'll look into this and fix it.

**Comment by tush0618** [comment]
I have applied the changes in my **policies.p**y file.
Its working fine now.

When will it get merged to the "develop" branch???

Thanks,
Tushar

**Comment by btorresgil** [comment]
@tush0618 It's merged in develop now.  Thanks again for reporting this.  We'll release the fix in pandevice 0.5.1 soon.

**Comment by tush0618** [comment]
@btorresgil and @shinmog thanks for quickly looking into the issue and fixing it.

Cheers,
Tushar
